cli/src/lib/system-contract-change.ts

```diff
@@ -17,7 +19,19 @@ export class SystemContractChange {
   }
 
   async downloadCurrentCode(client: BlockExplorerClient): Promise<ContractData> {
-    return client.getSourceCode(this.address);
+    try {
+      return await client.getSourceCode(this.address);
+    } catch (e) {
+      // Some system contracts do not have the code available in the block explorer. But this is not an error.
+      // For these contracts we cannot show the current source code.
+      if (e instanceof ContracNotVerified) {
+        const content = {
+          content: `Code for contract ${this.address} (${this.name}) is not available in block explorer`,
+        };
+        return new ContractData(this.name, { "messate.txt": content }, this.address);
+      }
+      throw e;
+    }
```

Comment:
Another polemic error handling. Again, I believe this is correct, even when it sounds weird.